src/Lab/GammaFun/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Silk.NET.Core.Native;
+using Silk.NET.GLFW;
+
+namespace GammaFun
+{
+    unsafe class Program
+    {
+        static void Main(string[] args)
+        {
+            var glfw = Glfw.GetApi();
+
+            glfw.Init();
+
+            var monitor = glfw.GetPrimaryMonitor();
+            
+            var window = glfw.CreateWindow(200, 200, "Gamma Test", null, null);
+
+            if (window == default)
+            {
+                glfw.Terminate();
+                return;
+            }
+
+            var ramp = glfw.GetGammaRamp(monitor);
+            var size = (int)(ramp->Size * sizeof(short));
+
+            var red = (short*)SilkMarshal.Allocate(size);
+            var green = (short*)SilkMarshal.Allocate(size);
+            var blue = (short*)SilkMarshal.Allocate(size);
```

Comment:
Afaik GLFW copies everything once you send the ramp over, so these can be stackallocs.

Reply:
But isn't that an unbound stackalloc and may cause a stackoverflow?

Reply:
Technically yes, but in practice this is usually a pretty small number. I don't ever see this going above 256 bytes allocated, to be honest. But your call.

Reply:
Fair enough, will change soon